newIDE/app/src/ResourcesList/ResourceSelector.js

```diff
@@ -191,6 +192,8 @@ export default class ResourceSelector extends React.Component<Props, State> {
           fps: 0,
           name: 'Image',
           isLooping: false,
+          metadata,
+          isTiled: true,
```

Comment:
You shuold not hardcode an object name here, as there are other objects that use a single image (particle emitter, panel sprite, etc...).
Instead, this information should either be generic or deduced from something else. The question is "why do we need this "isTiled" boolean?".
I guess it's because you want at the end to know if you want to store the metadata in the single resource, or pass it back to the editor in case you're editing multiple resources.

In this case, better name it "supportMultipleResources" or something like this.

Reply:
I initially decided to use it to determine if piskel is being called to edit a tiledSprite - in that case piskel-main will automatically switch piskel to tiled sprite view mode - which makes it easier to discover that piskel feature and have gdevelop guess the user's intent. But I can use isSingleFrame instead. The name being 'Image' has been there before - I didn't add it in this pull, but we can change it to something else if you want to :)

Reply:
I will remove it. singleImage basically does the same thing and for not I do not need to have two different values